app/config/sculpin_site.yml

```
@@ -58,6 +58,9 @@ subspecies:
     phpclasses:
         common: "PHP Classes"
         latin: "decuriae"
+    phpconjapan:
+        common: "PHPCON Japan"
+        latin: "iaponicacolloquium"
```

I think this fits the naming pattern better:
```suggestion
        latin: "colloquiumiapona"
```